notebooks/10_text_data.ipynb

```diff
@@ -0,0 +1,680 @@
+{
```

Comment:
<p>lower case -&gt; lowercase</p><p> </p><p>(I have read that both spellings are acceptable, though "lowercase" is definitely more common in US English, hence my suggestion.)</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/stijnvanhoey/pandas-getting-started-tutorials/pull/14/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='4'/>